test/conformance/ptp/ptp.go

```diff
@@ -364,6 +365,54 @@ var _ = Describe("[ptp]", func() {
 				Expect(slavePodDetected).ToNot(BeFalse(), "No slave pods detected")
 			})
 		})
+
+		Context("Running with fifo scheduling", func() {
+			BeforeEach(func() {
+				if discoveryFailed {
+					Skip("Failed to find a valid ptp slave configuration")
+				}
+				ptpPods, err := client.Client.Pods(PtpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})
```

Comment:
You should run this only on slave pods, right?
if podRole(pod, slaveNodeLabel) {

Reply:
The test isn't filtering whether the ptpconfig is a master or slave config, so should run on all ptp nodes.

Reply:
I am not sure, I will try to run this and see but I think the below code doesn't filter non-ptp enabled pods.
ptpPods, err := client.Client.Pods(PtpLinuxDaemonNamespace).List(context.Background(), metav1.ListOptions{LabelSelector: "app=linuxptp-daemon"})

Reply:
I was able to test this, marking as resoved